docs/cloud/getting-started-with-airbyte-cloud.md

```diff
@@ -8,7 +8,7 @@ To use Airbyte Cloud:
 
 1. If you haven't already, [sign up for Airbyte Cloud](https://cloud.airbyte.io/signup?utm_campaign=22Q1_AirbyteCloudSignUpCampaign_Trial&utm_source=Docs&utm_content=SetupGuide). 
 
-    Airbyte Cloud offers a 14-day free trial with $1000 worth of [credits](core-concepts.md#credits), whichever expires first. For more information, see [Pricing](https://airbyte.com/pricing).
+    Try Airbyte Cloud for free. For more information, see [Pricing](https://airbyte.com/pricing).
```

Comment:
I think this should say "Airbyte Cloud offers a 14-day free trial. For more information, see Pricing"
Context: I know we discussed keeping things generic but this feels too generic 😄

Reply:
Sounds good! I'll change it :)